docs/install_guides/installing-model-dev-tools.md

```diff
@@ -1,147 +1,205 @@
 # Install OpenVINO™ Development Tools {#openvino_docs_install_guides_install_dev_tools}
 
-If you want to download, convert, optimize and tune pre-trained deep learning models, install OpenVINO™ Development Tools, which provides the following tools:
+OpenVINO Development Tools is a set of utilities that make it easy to develop and optimize models and applications for OpenVINO. It provides the following tools:
 
 * Model Optimizer
 * Benchmark Tool
 * Accuracy Checker and Annotation Converter
 * Post-Training Optimization Tool
 * Model Downloader and other Open Model Zoo tools
 
-> **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI. 
+The instructions on this page show how to install OpenVINO Development Tools. If you are a Python developer, it only takes a few simple steps to install the tools with PyPI. If you are developing in C++, OpenVINO Runtime must be installed separately before installing OpenVINO Development Tools.
 
-## For Python Developers
+In both cases, Python 3.6 - 3.9 need be installed on your machine before starting.
 
-If you are a Python developer, you can find the main steps below to install OpenVINO Development Tools. For more details, see <https://pypi.org/project/openvino-dev>.
+> **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI.
 
-While installing OpenVINO Development Tools, OpenVINO Runtime will also be installed as a dependency, so you don't need to install OpenVINO Runtime separately.
+## <a name="python-developers"></a>For Python Developers
 
-### Step 1. Set Up Python Virtual Environment
+If you are a Python developer, follow the steps in the <a href="openvino_docs_install_guides_install_dev_tools.html#install-dev-tools">Installing OpenVINO Development Tools</a> section on this page to install it. Installing OpenVINO Development Tools will also install OpenVINO Runtime as a dependency, so you don’t need to install OpenVINO Runtime separately. This option is recommended for new users.
```

Comment:
```suggestion
If you are a Python developer, follow the steps in the <a href="o#install-dev-tools">Installing OpenVINO Development Tools</a> section on this page to install it. Installing OpenVINO Development Tools will also install OpenVINO Runtime as a dependency, so you don’t need to install OpenVINO Runtime separately. This option is recommended for new users.
```

Reply:
> 

I do not follow this change, could you elaborate it, please?

Reply:
openvino_docs_install_guides_install_dev_tools.html is the current file, right? we don't need to add the file name when referencing to an anchor in the same file

Reply:
Well, normally yes. We will see if it works during the tests ;)